appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/properties/YoungAndroidNavigationMethodChoicePropertyEditor.java

```diff
@@ -0,0 +1,29 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
```

Comment:
Because this is a new editor, you don't need to include the Google copyright line, and for MIT it should only be for 2019.